lib/minitest/reporters/spec_reporter.rb

```diff
@@ -42,11 +60,12 @@ def after_suite(_suite)
         puts
       end
 
-      def record_print_status(test)
-        test_name = test.name.gsub(/^test_: /, 'test:')
-        print pad_test(test_name)
-        print_colored_status(test)
-        print(" (%.2fs)" % test.time) unless test.time.nil?
+      def print_failure(test)
+        puts
+        puts test.klass
+        record_print_status(test)
+        print_info(test.failure, test.error?)
+        puts "Location:\n\t #{test.source_location.join(':')}"
```

Comment:
Hmm what value does printing the `source_location` add since we already print the backtrace?